_posts/2017-07-05-jenkins-job-builder.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Introduction to Jenkins Job Builder
+author: Elijah DeLee
+tags:
+  - pulpqe
+---
+
+The power of Jenkins
+----------------
+
+I first was exposed to Jenkins when I was working for a open source software
+project out of my university that wanted to use continuous integration as a
+part of their workflow. While services like [Travis](https://travis-ci.org/)
+and [Semaphore](https://semaphoreci.com/) can be
+excellent if you can accomplish your build within the constraints of the
+systems they provide, they could not meet the needs of that project. Enter
+[Jenkins](https://jenkins.io/), "the leading open source automation server".
+
+Jenkins is an powerful tool, and much of this power comes from the fact that it
+is so incredibly customizable. Configuring jobs via the web UI that Jenkins
+provides leaves many things to be desired, however. First, the task is fraught
+with human error. It is very easy to forget to expand that one nested menu and
+set a critical value, perhaps creating a faulty job that leads to misleading
+results or a broken CI/CD pipeline causing a headache for all involved, slowing
+work down instead of speeding it up (that's what automation is _supposed_ to
+do, right?). Secondly, once you have constructed your house of cards, you have
+only to experience a hard drive failure to lose all your work, or much more
+likely, something changes in your set up that leads you to hand updating dozens
+of jobs so carefully configured with hours of mouse clicks.
+
+Jenkins Job Builder (JJB)
+----------------
+
+To scale the process of Jenkins job definition it become obvious that one must
```

Comment:
s/become/became

This sentence needs to be broken up a bit or rewritten to be shorter/multiple sentences.